StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1121UnitTests.cs

```diff
@@ -0,0 +1,724 @@
+﻿namespace StyleCop.Analyzers.Test.ReadabilityRules
+{
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CodeFixes;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using StyleCop.Analyzers.ReadabilityRules;
+    using TestHelper;
+    using System;
+
+
+    /// <summary>
+    /// This class contains unit tests for <see cref="SA1121UseBuiltInTypeAlias"/>
+    /// </summary>
+    [TestClass]
+    public class SA1121UnitTests : CodeFixVerifier
+    {
+        static readonly Tuple<string, string>[] _referenceTypes = new Tuple<string, string>[]
+        {
+            new Tuple<string,string>("object", "Object"),
```

Comment:
`nameof(Object)` for the right hand side here. :wink: The same for the items below.